packages/ui/src/TapToEdit.tsx

```diff
@@ -33,7 +33,7 @@ const TapToEditTitle = ({
   );
   if (showDescriptionAsTooltip) {
     return (
-      <Tooltip idealDirection="top" text={description}>
+      <Tooltip idealPosition="top" text={description}>
```

Comment:
Thanks for going through and standardizing these!

Reply:
Don't thank me too much... this is necessary to pass test 😅